### ISSUE TITLE
UTF offset wrong time

### ISSUE THREAD

**Issue by fmunozs** [o0shojo0o/PixelIt#73]
I'm on Bogota (UTC -5) but when I pick -5 on the offset , the clock says it's 13:21 while the time is 12:21

![image](https://user-images.githubusercontent.com/1041604/133828831-dcfcc0c2-b174-48c8-a28f-423b4671802f.png)

**Comment by o0shojo0o** [comment]
Yes the error I can confirm, since I have hin in Germany the summer and winter time, I calculate automatically always one hour up or down. 
I just did not think that not the whole world has to admit the stupid summer and winter time changeover.  

I will come up with something for the next update :o)
